packages/salesforcedx-lightning-lsp-common/src/index.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+export {
+  toResolvedPath,
+  isLWCRootDirectoryCreated,
+  relativePath,
+  pathStartsWith,
+  getExtension,
+  getBasename,
+  getSfdxResource,
+  memoize,
+  readJsonSync,
+  writeJsonSync,
+  extractJsonFromImport,
+  SfdxTsConfig,
+  TsConfigPaths
+} from './utils';
+
+// Re-export from base-context
+export {
+  BaseWorkspaceContext,
+  Indexer,
+  AURA_EXTENSIONS,
+  processTemplate,
+  getModulesDirs,
+  updateForceIgnoreFile
+} from './baseContext';
+
+// Re-export from shared
+export { WorkspaceType, isLWC, getSfdxProjectFile, detectWorkspaceHelper, detectWorkspaceType } from './shared';
+
+// Re-export from indexer
+export {
+  TagInfo,
+  createTagInfo,
+  getAttributeInfo,
```

Comment:
it's worth reviewing whether these are used somewhere.  I think it escapes `knip` because it's reexported and assumed to part of the packagek's API, but if nothing in our repo is using ex: getAttributeInfo then it can be removed.